### PR TITLE
#145 get rid of delete time in organization table

### PR DESCRIPTION
**Pull request by kostamir**

### CODE REVIEW

File: src/sql-queries/organization.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { QueryConfig } from 'pg';
-import { select, insert, update } from 'sql-bricks-postgres';
+import { select, insert, update, delete as remove } from 'sql-bricks-postgres';
```
Comment (AzisK) [inline src/sql-queries/organization.ts:2]: Why rename delete to remove?

File: src/sql-queries/organization.ts [hunk 2]
```diff
@@ -21,36 +21,20 @@ interface UpdateOrganizationInput {
...
         .toParams();
 
 export const getOrganizationsQuery = (): QueryConfig =>
-    select()
-        .from('organization')
-        .where({
-            delete_time: null,
-        })
-        .toParams();
+    select().from('organization').where({}).toParams();
```
Comment (AzisK) [inline src/sql-queries/organization.ts:31]: I believe we can skip the `where` part?

**Review by AzisK — CHANGES_REQUESTED** [review]
Well done. It looks good. Left a comment for a possible improvement and a question

File: src/sql-queries/organization.ts (hunk 2 above)
Comment (kostamir) [reply src/sql-queries/organization.ts]: Done.

File: src/sql-queries/organization.ts (hunk 1 above)
Comment (kostamir) [reply src/sql-queries/organization.ts:2]: Tried to use delete as is but vscode thew an error so we came up with this workaround.

**Review by AzisK — APPROVED** [review]

File: src/sql-queries/organization.ts (hunk 1 above)
Comment (JBK91) [reply src/sql-queries/organization.ts:2]: Because delete is reserved JS keyword

**Review by ignasposka — APPROVED** [review]

**Pull request merged**